pyproject.toml

```diff
@@ -45,6 +45,7 @@ odmantic = { version = "*", optional = true }
 pydantic = { version = "*", optional = true, extras = ["email"] }
 typing-extensions = "*"
 attrs = {version = ">=22.2.0", optional = true}
+sqlalchemy = { version = "*", optional = true }
```

Comment:
I think this can be removed. `sqlalchemy` isn't officially supported yet.

Reply:
We need to add `sqlalchemy` to `pyproject.toml` somewhere so that we can run the new tests

Putting it in the dev dependencies section felt a bit strange, but perhaps that makes more sense than having it as an optional extra?

If we go with that approach, would it make sense to pin the dependency to SQLAlchemy >=2 for now? Or would you prefer an explicit pytest marker (probably based on `from sqlalchemy import __version__`)

Reply:
I think keeping it in dev dependencies is fine. It's only used for testing as of now. Also, I think it would be better to have an explicit pytest mark.